.github/workflows/ci.yml

```
@@ -21,14 +21,22 @@ jobs:
 
             echo "::group::Run simulation"
             chmod +x tests/run_ODD_sim.sh
-            ./tests/run_ODD_sim.sh
\ No newline at end of file
+            ./tests/run_ODD_sim.sh
+      - uses: actions/upload-artifact@v4
+        with:
+          name: sim_output
+          path: tests/run/
+          if-no-files-found: error
+          retention-days: 7
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)